install-template.sh

```diff
@@ -302,21 +307,32 @@ validate_opt () {
 
 absolutify() {
     local file_path="$1"
-    local file_path_dirname="$(dirname "$file_path")"
-    local file_path_basename="$(basename "$file_path")"
-    local file_abs_path="$(abs_path "$file_path_dirname")"
+    local file_path_dirname="${file_path%/*}"
+    local file_path_basename="${file_path##*/}"
```

Comment:
NB: These `dirname`/`basename` substitutions assume more well-formed paths, e.g., no trailing slashes etc.
Let me know if you aren't comfortable with adding these constraints and if I should revert to `dirname`/`basename`.